app.js

```diff
@@ -1,25 +1,43 @@
 const express = require('express');
-
 const hbs = require('hbs');
 const path = require('path');
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
-
 const app = express();
 const punkAPI = new PunkAPIWrapper();
 
+hbs.registerPartials(__dirname + "/views/partials");
+
 app.set('view engine', 'hbs');
 app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
-// Register the location for handlebars partials here:
+app.get('/', (req, res) => {
+  res.render('index');
+});
 
-// ...
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers().then(beersFromApi =>
+    res.render('beers', {
+      beersFromApi
+    })
+  ).catch(error => console.log(error));
+});
 
-// Add the route handlers here:
+app.get('/random-beer', (req, res) => {
+  punkAPI.getRandom().then(randomBeer =>
+    res.render('random-beer', {
+      randomBeer
+    })
+  ).catch(error => console.log(error))
+});
 
-app.get('/', (req, res) => {
-  res.render('index');
+app.get(`/beers/beer-:id`, function (req, res) {
+  punkAPI.getBeer((req.params).id).then(beerID => res.render(`beer-by-id`, {
+    beerID
+  })).catch(error => error)
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.listen(3000);
```

Comment:
It's considered good practice to have a callback to console.log which port your server is running on, I also recommend you get into the habit of storing your port number in a variable (it's commonly done as well)
```
const PORT = 3000;

app.listen(PORT, () => console.log('Server is running on port ${PORT}'));

Reply:
Thanks! Will make it a habit